docker/Dockerfile.juicefsruntime

```diff
@@ -16,14 +16,19 @@ RUN apk add --update curl tzdata iproute2 bash libc6-compat vim &&  \
  	cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && \
  	echo "Asia/Shanghai" >  /etc/timezone
 
-RUN curl -o helm-v3.0.3-linux-amd64.tar.gz http://aliacs-k8s-cn-hongkong.oss-cn-hongkong.aliyuncs.com/public/pkg/helm/helm-v3.0.3-linux-amd64.tar.gz && \
+RUN if [ `uname -m` == "aarch64" ] ; then curl -o helm-v3.0.3-linux-arm64.tar.gz https://get.helm.sh/helm-v3.7.2-linux-arm64.tar.gz && \
```

Comment:
How about using `https://get.helm.sh/helm-v3.0.3-linux-arm64.tar.gz` ?

Reply:
Amended the PR to use helm v3.0.3 for ARM64.